gatsby-node.js

```diff
@@ -112,8 +122,8 @@ exports.sourceNodes = async (
       collections: collectionsArray,
       files: filesArray,
       mainImageHref,
-      parent: null,
-      children: [],
+      parent: parentId,
+      children: childrenArray,
```

Comment:
Will the fact that `parentId` and `childrenArray` _could_ be `undefined` now potentially cause any errors?

Reply:
It hasn't on our store.  i've added fallback values in case there are cases we're not hitting that would break